query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/alreadyConverted/NestedCreateMutationInsideUpdateSpec.scala

```diff
@@ -9,61 +9,6 @@ class NestedCreateMutationInsideUpdateSpec extends FlatSpec with Matchers with A
 
   //todo which tests to keep and which ones to delete???? Some do not really test the compound unique functionality
 
-  "a P1! to C1! relation" should "error since old required parent relation would be broken" in {
```

Comment:
This looks like a dedicated test case for required 1:1 relations that we don't need anymore.

Reply:
Jep, that can go.